config-example/cl/config.yaml

```diff
@@ -5,12 +5,12 @@ CONFIG_NAME: testnet # needs to exist because of Prysm. Otherwise it conflicts w
 # Genesis
 # ---------------------------------------------------------------
 # `2**14` (= 16,384)
-MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 16384
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: $MIN_GENESIS_ACTIVE_VALIDATOR_COUNT
 # Mar-01-2021 08:53:32 AM +UTC
 # This is an invalid valid and should be updated when you create the genesis
-MIN_GENESIS_TIME: 0
-GENESIS_FORK_VERSION: 0x13001020
-GENESIS_DELAY: 120
+MIN_GENESIS_TIME: $CL_TIMESTAMP
```

Comment:
@parithosh This env var does not appear in the values.env. Is that expected?

Reply:
Ah, I think i need to rename it to `GENESIS_TIMESTAMP` to keep it standard. Thank you!